### PR TITLE
Makes html fields non editable and disables inline edit box in studio

### LINKED ISSUE #3400: HTML field can't be edited
#### Issue

I can add an HTML field, place it into my edit and detail view but whilst editing data, there is no field showing up.

#### Expected Behavior

Having a field for my HTML content

#### Actual Behavior

html_field
No HTML field is being shown.

#### Steps to Reproduce

1. Add an HTML field (tested 3 modules so far - everywhere the same)
2. Place it in edit/detail view
3. Try to edit one entry and now HTML field will show up
4. (if there is default text inside from the field creation, it shows on the detail view (but can't be edited)

#### Your Environment

* SuiteCRM Version used: 7.8.1, 7.8.3, 7.7.9
* Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)): Chrome, Firefox
* Environment name and version (e.g. MySQL, PHP 7): Apache, NGinx, PHP 5.6, PHP7, MySQL
* Operating System and version (e.g Ubuntu 16.04): Debian, Ubuntu
Tested on 3 different servers - all the same.

### LINKED ISSUE #4957: Html fields showing the string value of the html, not the html itself
Html fields are to show html. The are showing the string value of the html, not the html itself. The literal string is being shown on version 7.9.9

Before there was nothing being shown on Edit and DetailViews. The update now shows the literal string representation of the fields html.

Studio shows the HTML in the edit of the field as the display value of the HTML like you would in a web browser.

#### Your Environment

* SuiteCRM Version used: 7.9.9
* Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)):
* Environment name and version (e.g. MySQL, PHP 7):
* Operating System and version (e.g Ubuntu 16.04):

### PR DESCRIPTION
**Pull request by PedroErnst**
Reverses a recent change which made html fields editable / not display their static content.

## Description

Html fields are designed just to show pre defined html, not to be editable.
With this change html fields will display their static value in detail and editview, and inline editing will be disabled by default.

## Motivation and Context

Fixes #4957 
Fixes #3400 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code follows the code style of this project found [here](https://docs.suitecrm.com/community/contributing-code/coding-standards/).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://docs.suitecrm.com/community/contributing-code/) guidelines.

### CODE REVIEW

**Comment by Dillon-Brown** [comment]
Assessed

File: include/SugarFields/Fields/Html/EditView.tpl [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{*
+/**
+ *
+ * SugarCRM Community Edition is a customer relationship management program developed by
```
Comment (horus68) [inline include/SugarFields/Fields/Html/EditView.tpl]: IF this is a new file it do not require the full copyright notice as this is not a SugarCRM forked file but a new file.

Comment (PedroErnst) [reply include/SugarFields/Fields/Html/EditView.tpl]: First time I hear this, not saying it isn't true but I've seen the notice included in every new file so far...

https://github.com/salesagility/SuiteCRM/pull/5669/files
https://github.com/salesagility/SuiteCRM/pull/5735/files

Just as examples... there is many more.

Comment (horus68) [reply include/SugarFields/Fields/Html/EditView.tpl]: I know its the actual way of doing things but it should be discussed.
Else what will be like in 10 years: plenty of Sugar notices on files not created with Sugar code! 
See a RFC https://github.com/salesagility/SuiteCRM/issues/5952

Comment (Dillon-Brown) [reply include/SugarFields/Fields/Html/EditView.tpl]: I'm not entirely certain but it's my understanding that any modifications to core (even if it's a new file) would "inherit" the AGPL and thus require the license. Definitely something to investigate / discuss further though!

Comment (horus68) [reply include/SugarFields/Fields/Html/EditView.tpl]: Any file inherit the AGPL but not the "Appropriate Legal Notices" or Sugar references on all files after a fork.
For legal you can only say "licensed under AGPL" in the header and provide a full text copy as a file. Take a look into Joomla that was a fork from Mambo.
The GPL doesn't require advertisement, just disclosure
Anyway this is a Legal issue and should be taken in detail by SalesAgility.

Interesting light reading: https://opensource.stackexchange.com/questions/4971/what-are-the-attribution-requirements-of-the-agpl-license

**Pull request merged**